Components/modal/modal.html

```diff
@@ -150,7 +156,7 @@ <h4>Shop with exclusive offers</h4>
                                                 sleeveless,
                                                 button closures</p>
                                         </div>
-                                        <div class="CTA-Container">
+                                        <div class="CTA-Container jcc">
```

Comment:
Try to give a more meaningful name to the class. As a user, it's quite confusing to understand what's jcc.

Reply:
check the CSS file I have mentioned proper comments for atomic CSS!